src/main/java/org/eolang/compiler/syntax/Type.java

```diff
@@ -61,9 +64,9 @@
      * Convert it to Java file (path, content).
      * @return Java code
      */
-    public Map.Entry<String, String> java() {
+    public Map.Entry<Path, String> java() {
         return new AbstractMap.SimpleEntry<>(
-            this.name,
+            Paths.get(String.format("%s.java", this.name)),
```

Comment:
trivial as this may be I feel like all the string formatting should be in it's own class. It will hide the implementation and give one place to reuse and maintain.